cmd/jaeger/config-elasticsearch.yaml

```diff
@@ -20,10 +20,35 @@ extensions:
     backends:
       some_storage:
         elasticsearch:
-          index_prefix: "jaeger-main"
+          indices:
+            spans:
+              prefix: "jaeger-main"
+              date_layout: "2006-01-02"
+              rollover_frequency: "day"
+              shards: 5
+              replicas: 1
+            services:
+              prefix: "jaeger-main"
+              date_layout: "2006-01-02"
+              rollover_frequency: "day"
+              shards: 5
+              replicas: 1
+            dependencies:
+              prefix: "jaeger-main"
+              date_layout: "2006-01-02"
+              rollover_frequency: "day"
+              shards: 5
+              replicas: 1
+            sampling:
+              date_layout: "2006-01-02"
+              rollover_frequency: "day"
+              shards: 5
+              replicas: 1
       another_storage:
         elasticsearch:
-          index_prefix: "jaeger-archive"
+          indices:
+            spans:
```

Comment:
Q: is the services index being built at all when the storage is in the archive mode? I don't think it should be, but in this configuration `services.prefix: "jaeger-main"` would be the default, and the archive storage is not told in any way that it is indeed archive.

Reply:
@yurishkuro my apologies i don't have a lot of context here but how should we proceed with this?

Reply:
1. we need to check in the code if writing/reading to/from Archive storage relies exclusively on spans index, or if it uses any other indices. If the latter, then we need to override the prefix to `jaeger-archive` not just for spans but for other indices too. This is where the `defaults:` section I mentioned in another comment would be useful - we could just define it once instead of duplicating all the information.